metricbeat/metricbeat.full.yml

```diff
@@ -64,6 +64,11 @@ metricbeat.modules:
   # if true, exports the CPU usage in ticks, together with the percentage values
   #cpu_ticks: false
 
+  # A list of filesystem types to ignore. The filesystem metricset will not
+  # collect data from filesystems matching any of the specified types and
```

Comment:
Add a comma before "and" to connect two independent clauses: `...any of the specified types, and`